shared/src/common/enums/validation/master/master.ts

```diff
@@ -1 +1,2 @@
 export { MasterSignUpValidationMessage } from './master-sign-up.validation-message';
```

Comment:
```suggestion
export { MasterValidationMessage } from './master-sign-up.validation-message';
```
rename filename & enum to the `MasterValidationMessage` and use it for both sign-up and sign-in